src/components/SplashScreen.js

```diff
@@ -1,62 +1,64 @@
-import { Box } from '@mui/system';
-import React, { useState, useEffect } from 'react';
-import { useNavigate } from 'react-router-dom';
-import logo from '../assets/img/teamMX-logo.png';
-import '../assets/css/style.css';
-import {ClipLoader} from 'react-spinners';
-import { Button, ButtonGroup, Slide, Typography } from '@mui/material';
+import { Box } from "@mui/system";
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import logo from "../assets/img/teamMX-logo.png";
+import "../assets/css/style.css";
+import { ClipLoader } from "react-spinners";
```

Comment:
What does this look like? MUI has spinners built in, so u want to justify using an external library

Reply:
Changed react-spinners to MUI Circular Progress